activevalidators.gemspec

```diff
@@ -19,7 +19,7 @@ Gem::Specification.new do |s|
   s.add_dependency 'date_validator'
   s.add_dependency 'activemodel'            , '>= 3.0'
   s.add_dependency 'phony'                  , '~> 2.0'
-  s.add_dependency 'countries'              , '~> 1.2'
+  s.add_dependency 'countries'              , '~> 1.2', '< 3.0'
```

Comment:
This means "greater than 1.2 AND less than 2 AND less than 3", which isn't really what you intended, I think.

Maybe you meant something like this?

```ruby
s.add_dependency 'countries', '>= 1.2', '< 3.0'
```

Reply:
Sure. my bad :/ i will fix